notifier/orb.yaml

```diff
@@ -1,12 +1,56 @@
 version: 2.1
 description: "Tools for notifying external services about CI events"
 
+orbs:
+  slack: circleci/slack@4.7.0
+
 executors:
   alpine-stable:
     docker:
     - image: alpine:latest
     resource_class: small
 
+commands:
+  slack-on-deploy:
+    description: |
+      Uses slack orb to send a notification to slack when a deployment fails.
+    parameters:
+      channel:
+        default: ${SLACK_DEFAULT_CHANNEL}
+        description: |
+          Destination channel for your slack messages.
+        type: string
+    steps:
+      - slack/notify:
+          channel: <<parameters.channel>>
+          custom: |
+            {
+              "text": "",
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "*Failure* :x: \n - Job name: `${CIRCLE_JOB}` \n - Repository name: `${CIRCLE_PROJECT_REPONAME}` \n - Branch `${CIRCLE_BRANCH}` \n - Blame :point_right: @${CIRCLE_USERNAME} \n - Build number: #${CIRCLE_BUILD_NUM}"
```

Comment:
This section can actually be split into multiple sections for better formatting and ease of reading. Instead of using `\n` to create sections, you can add multiple entries under `fields` like in https://github.com/CircleCI-Public/slack-orb/blob/master/src/message_templates/basic_fail_1.json.

Reply:
I think we can remove the "build number" here as well -- that would only really be helpful in tracking down the CI job, but your button for the build url should already handle that.

Reply:
@TheKevJames I removed the build number.
@jonathan-johnston I didn't know that I can do something like that. I'll go for that.